src/new-components/teaser/teaser.tsx

```diff
@@ -0,0 +1,181 @@
+`;
+
+const TeaserText = styled.div`
+  ${TextStyles.textR};
+  margin: 0;
+`;
+
+const StyledTeaserTitle = styled.p<{
+  large: boolean;
+  hasTopline: boolean;
+}>`
+  ${TextStyles.headlineS}
+  ${(props) => props.large && TextStyles.headlineM};
+
+  margin-top: 0;
+  margin-bottom: ${(props) => (props.hasTopline ? '8px' : '16px')};
+`;
+
+const StyledIcon = styled(Icon)`
+  margin-top: 14px;
+`;
+
+export interface TeaserProps {
+  /**
+   * The main title of the teaser
+   */
+  title: string;
+  /**
+   * A topline/kicker which can be added
+   */
+  topline?: string;
+  /**
+   * An already formatted date string which is mainly used for the blog post overview
+   */
+  dateFormatted?: string | null;
+  /**
+   * The image above the title. You can't pass in an image and illustration. If you pass in an image and illustration the illustration will be taken.
+   */
+  image?: JSX.Element;
+  /**
+   * An illustration for the teaser
+   */
+  illustration?: IllustrationType;
+  /**
+   * Adds a link to another page and makes the whole teaser clickable
+   */
+  linkTo?: string;
+  className?: string;
+  /**
+   * The main text of the teaser is entered as child
+   * */
+  children: React.ReactNode | React.ReactNode[];
+}
+
+const ConditionalLink = ({ to, children, onHover }) => {
+  if (to) {
+    return (
+      <Link
+        to={to}
+        onMouseOver={() => {
+          onHover(true);
+        }}
+        onMouseLeave={() => onHover(false)}
+      >
+        {children}
+      </Link>
+    );
+  }
+
+  return children;
+};
+
+export const Teaser = ({
+  topline,
+  title,
+  dateFormatted,
+  image,
+  linkTo,
+  illustration,
+  className,
+  children,
+}: TeaserProps): JSX.Element => {
+  const [hoverActive, setIsHoverActive] = useState<boolean>(false);
+  const hasToplineContainer = Boolean(topline || dateFormatted);
+
+  return (
+    <TeaserContainer className={className} hover={hoverActive}>
+      <ConditionalLink
+        onHover={(hasHover) => setIsHoverActive(hasHover)}
+        to={linkTo}
+      >
+        {!illustration && image && (
+          <ImageContainer hover={hoverActive}>{image}</ImageContainer>
+        )}
+        {illustration && (
+          <StyledIllustration
+            show={illustration}
+            size={IllustrationSize.MEDIUM}
+            hover={hoverActive}
+          />
+        )}
+        {hasToplineContainer && (
+          <ToplineContainer>
+            {topline && <Topline>{topline}</Topline>}
+            {dateFormatted && (
+              <Timestamp hover={hoverActive}>{dateFormatted}</Timestamp>
+            )}
+          </ToplineContainer>
+        )}
+        <StyledTeaserTitle
+          large={!(image || illustration)}
+          hasTopline={hasToplineContainer}
+        >
+          {title}
+        </StyledTeaserTitle>
+        <TeaserText>{children}</TeaserText>
+        {linkTo && <StyledIcon show={'arrow_right'} />}
+      </ConditionalLink>
+    </TeaserContainer>
+  );
+};
```

Comment:
Seeing big components like that I always feel an impulse to refactor it.
1. image + illustration is a semantic block which you could handle in a local component
2.  topline with date is a separate part
3. the title, text & icon is another.

That way this part could look much more accessible for a developer. Please give it a try in a future PR 🙏
```
<Teaser>
   <Cover/>
  <Topline/>
  <Content/>
</Teaser>
```